.travis.yml

```diff
@@ -27,6 +27,7 @@ cache:
   directories:
     - $HOME/.ivy2/cache
     - $HOME/.sbt
+    - $HOME/.coursier/cache
```

Comment:
We are not using coursier in this repo. Why we should cache it then?

Reply:
We totally are. I just didn't know which directory to cache. Every time scalafmt runs it downloads stuff using coursier. Scalafix itself also uses coursier to download stuff.